util/Sequencer/Note.gd

```diff
@@ -1,6 +1,6 @@
 class_name Note extends Resource
 
 var duration: float = 0
```

Comment:
let duration be a delta too. ie. instead of duration use note_end_delta
```suggestion
var note_end_delta: float = 0
```

Reply:
Duration is clearer :sweat_smile:

Reply:
but everywhere we are converting duration to note end delta

Reply:
The problem with naming it end_delta is, what is it relative to? The previous note's start or the current note's start? Making it duration means the note started after a delta time from the previous note's start, and it runs for the duration before stopping

Reply:
oof true